examples/covid19-stat-publisher/processor/processor.bal

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/ftp;
+import ballerina/io;
+import ballerina/log;
+import ballerina/task;
+import ballerina/file;
+import ballerinax/rabbitmq;
+
+public string[] newFiles = [];
+
+listener ftp:Listener secureRemoteServer = check new({
+    protocol: ftp:SFTP,
+    host: "localhost",
+    auth: {
+        credentials: {
+            username: "wso2",
+            password: "wso2123"
+        },
+        privateKey: {
+            path: "../resources/keys/sftp.private.key",
+            password: "changeit"
+        }
+    },
+    port: 21213,
+    path: "",
+    pollingInterval: 2,
+    fileNamePattern: "(.*).csv"
+});
+
+service "ftpServerConnector" on secureRemoteServer {
+    function onFileChange(ftp:WatchEvent event) {
+        foreach ftp:FileInfo addedFile in event.addedFiles {
+            int startIndex = (addedFile.path.lastIndexOf("/") ?: 0) + 1;
+            int lastIndex = addedFile.path.length();
+            string fileName = addedFile.path.substring(startIndex, lastIndex);
+            log:printInfo("Added file path: " + fileName);
+            stream<byte[] & readonly, io:Error?>|error fileStream = sftpClient->get(fileName);
+            if fileStream is stream<byte[] & readonly, io:Error?> {
+                checkpanic io:fileWriteBlocksFromStream(fileName, fileStream, option = io:APPEND);
+                checkpanic fileStream.close();
+                newFiles[newFiles.length()] = fileName;
+            } else {
+                io:println(fileStream);
```

Comment:
I think we should use the log module here.

Reply:
Changed all the related instances.